README.md

```diff
@@ -39,12 +39,25 @@ Custom Resource Definitions (CRDs) and Kubernetes-native tooling.
 You can use [Minikube or Kind][k8s-io-tools] on your local machine or use
 a hosted Kubernetes service like [GKE](https://cloud.google.com/kubernetes-engine/).
 
-Setting up Kong for Kubernetes is as simple as:
+### Install the Gateway API CRDs
+
+This command will install all resources that have graduated to beta,
+including GatewayClass, Gateway, ReferenceGrant, and HTTPRoute.
```

Comment:
```suggestion
This command will install all the GA resources,
including `GatewayClass`, `Gateway`, `ReferenceGrant`, and `HTTPRoute`.
```

Reply:
Thanks! But the `ReferenceGrant` still in v1beta1
https://github.com/kubernetes-sigs/gateway-api/blob/52c2994ed9de1c287a37465490b91cfcf01bf16e/apis/v1beta1/referencegrant_types.go#L43

Reply:
```suggestion
This command will install all resources that have graduated to beta,
including `GatewayClass`, `Gateway`, `ReferenceGrant`, and `HTTPRoute`.
```

Reply:
I referred to the description here https://github.com/kubernetes-sigs/gateway-api/blob/52c2994ed9de1c287a37465490b91cfcf01bf16e/site-src/guides/index.md?plain=1#L38-L40

Reply:
How about "Resources graduated to GA or beta" here?

Reply:
It looks like that upstream description is a bit out of date. At the moment, Gateway, GatewayClass and HTTPRoute are v1, while ReferenceGrant is beta. +1 on Tao's suggestion.

Reply:
https://github.com/kubernetes-sigs/gateway-api/pull/2606